apps/web/app/api/social/og/image/route.tsx

```diff
@@ -115,6 +114,4 @@ async function handler(req: NextRequest) {
   }
 }
 
-const getHandler = apiRouteMiddleware(handler);
-
-export { getHandler as GET };
```

Comment:
- We run this API on Edge, and `import { performance } from "@calcom/lib/server/perfObserver";` in `defaultResponderForAppDir` can only run in Node.js.
- Ideally, we should modify this performance function, so that we can use the wrapper in APIs running on Edge too
- But it is not high priority, because in Pages Router, we weren't using `defaultResponder` for this API anyways